inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterNodeEntityMapper.xml

```diff
@@ -114,8 +114,8 @@
             </if>
             <if test="keyword != null and keyword != ''">
                 and (
-                ip like CONCAT('%', #{keyword}, '%')
```

Comment:
keyword is used for fuzzy queries.For an exact lookup, add
```
 <if test="ip != null and ip != ''">
      and ip = #{ip, jdbcType=VARCHAR}
 </if>
```

Reply:
> keyword is used for fuzzy queries.For an exact lookup, add
> 
> ```
>  <if test="ip != null and ip != ''">
>       and ip = #{ip, jdbcType=VARCHAR}
>  </if>
> ```

@fuweng11 It will cause agent exception when use like. So we should use =?

Reply:
Exact lookup cannot be used in keyword.